comp/dogstatsd/server/parse.go

```diff
@@ -188,7 +189,7 @@ func (p *parser) parseMetricSample(message []byte) (dogstatsdMetricSample, error
 	}
 
 	return dogstatsdMetricSample{
-		name:        p.interner.LoadOrStore(name),
+		name:        p.interner.LoadOrStore(name, "", nil),
```

Comment:
If `LoadOrStore()` is always called with `, "", nil)` is probably worth adding an helper that doesn't require those arguments

Reply:
The third PR in the series has the plumbing - that includes useful values for all 3 args.  Use that for reference: https://github.com/DataDog/datadog-agent/blob/lally/exp-mem-metrefs/comp/dogstatsd/server/parse.go#L190

Reply:
I think i'd be better to only add it when we need it no ?